code/modules/arcade/mob_hunt/game_server.dm

```diff
@@ -0,0 +1,164 @@
+var/global/datum/controller/process/mob_hunt/mob_hunt_server
+
+/datum/controller/process/mob_hunt
+	var/max_normal_spawns = 15		//change this to adjust the number of normal spawns that can exist at one time. trapped spawns (from traitors) don't count towards this
+	var/list/normal_spawns = list()
+	var/max_trap_spawns = 15		//change this to adjust the number of trap spawns that can exist at one time. traps spawned beyond this point clear the oldest traps
+	var/list/trap_spawns = list()
+	var/list/connected_clients = list()
+	var/server_status = 1			//1 is online, 0 is offline
+	var/reset_cooldown = 0			//number of controller cycles before the manual_reboot proc can be used again (ignored if server is offline so you can always boot back up)
+	var/obj/machinery/computer/mob_battle_terminal/red_terminal
+	var/obj/machinery/computer/mob_battle_terminal/blue_terminal
+	var/battle_turn = null
+
+/datum/controller/process/mob_hunt/setup()
+	name = "Nano-Mob Hunter GO Server"
+	start_delay = 20
+	mob_hunt_server = src
+
+/datum/controller/process/mob_hunt/Destroy()
```

Comment:
To be honest, admins *really* shouldn't be touching controllers, unless they know exactly what they're doing...even then, it's better to halt a controller from firing rather than...deleting it.